templates/default/install_info.yaml.erb

```diff
@@ -0,0 +1,6 @@
+<%= JSON.parse(({
```

Comment:
`JSON.parse().to_json.to_yaml` is this the way to print a yaml? 🤔

Reply:
C/p from other template, I'll improve